cpp/src/io/parquet/page_decode.cuh

```diff
@@ -122,7 +122,7 @@ struct null_count_back_copier {
  */
 constexpr bool is_string_col(PageInfo const& page, device_span<ColumnChunkDesc const> chunks)
 {
-  if (page.flags & PAGEINFO_FLAGS_DICTIONARY != 0) { return false; }
+  if ((page.flags & PAGEINFO_FLAGS_DICTIONARY) != 0) { return false; }
```

Comment:
Or just this:
```suggestion
  if (page.flags & PAGEINFO_FLAGS_DICTIONARY) { return false; }
```

Reply:
but that's implicit conversion and I don't like that :)